### PR TITLE
Feature:Final step

### PR DESCRIPTION
**Pull request by NickEmma**
Greetings 👋🏻
Thanks for taking the time to review my code

Changes implemented
✅ Create rails react single application
✅ Create Greeting API endpoint (API/v1/greeting)
✅ Create a Greeting route using React
✅ Create a redux store, action, and a reducer to fetch the data from the API ('/greeting')

### CODE REVIEW

File: app/views/api/v1/greetings/index.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1 @@
+json.text @greeting.text
```
Comment (GzimAsani) [inline app/views/api/v1/greetings/index.json.jbuilder:1]: - [ ] I am getting an error when visiting the index page, kindly update it image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @NickEmma           :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with implementing react router :+1: 

 ## Required Changes 


- [ ] Please add a professional readme by using the Microverse [template](https://github.com/microverseinc/readme-template)
- [ ] Check the comment below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/api/v1/greetings/index.json.jbuilder (hunk 1 above)
Comment (brytebee) [reply app/views/api/v1/greetings/index.json.jbuilder:1]: Just like this previous review, I am getting the same error.
image

Kindly research more about Jbuilder syntax and usage to render your views.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @NickEmma👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app/views/api/v1/greetings/index.json.jbuilder (hunk 1 above)
Comment (NickEmma) [reply app/views/api/v1/greetings/index.json.jbuilder:1]: @brytebee i am on windows and i specified it on the readme. it's working fine and i don't know why you requested for changes.

**Review by julie-ify — APPROVED** [review]
Hi @NickEmma,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**